### ISSUE TITLE
Can't setup 'Other' printer under Linux. Setup screen not large enough.

### ISSUE THREAD

**Issue by Lloyd-H1** [daid/LegacyCura#1493]
I went to setup a new 3D printer in Cura. It happens to be a Wanhoa i3. I get to the 1st list of printers in the Configuration Wizard and check 'Other' at the bottom. I get to the 2nd list of printers in the Configuration Wizard, and the bottom part of the list is not showing. The last printer that I can see is RigidBotBig. Everything under that is hidden. Nor is there any way to scroll down or expand the configuration window.

Running Cura 15.04.5 on Linux Mint 17.3

[cura2016june06](https://cloud.githubusercontent.com/assets/15940631/15815700/6ff5a7f0-2b94-11e6-8f62-4ffac8fda588.jpg)

**Comment by Lloyd-H1** [comment]
I just installed Cura 15.04.6 on my Linux Mint 17.3 computer. Still have the same problem. Can't install new "Other" printer. It won't scroll at all to show bottom of menu.

https://drive.google.com/open?id=0BwX7dDC8hfVTWHFmcld5ZnNuQjA

Also I ran into a problem uploading the screenshot. The Github Repository started creating a storage site for me. I am reasonably sure it is related to some recent GIT software that I installed. But I am not sure what.  I think that I installed some Github software that I have no use for. 
Anyway, I uploaded the photo to my Google drive and posted a link here.
